### PR TITLE
Story 2: Lobby UI Enhancement

### PR DESCRIPTION
**Pull request by varoongupta**
**Description:** The lobby page UI was initially designed as part of the minimum viable product, hence, there was little to no attention expended on the UI design of the page. Currently a lot of important information is presented on this page, such as the host and guests in the lobby, as well as several filter options for the host. This story enhances the UI to make it more appealing to the user.

**Dependencies:** None

**Subtasks:**

- [x] Update colour scheme in CSS to match application theme and consistency

- [x] Fix alignment and style of each aspect

**Justification:** This task requires planning a style for the whole application, and applying it to every screen to ensure consistency. Since this is a design task, it will require constant revision and may be subject to change frequently. Applying changes may require research and troubleshooting, earning this story 5 points.

Screen Shot 2020-12-05 at 9 45 46 PM

### CODE REVIEW

File: wewatch-client/src/components/Lobby/Lobby.css [hunk 1]
```diff
@@ -10,12 +14,24 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    background: #FFFFFF;
+    background: rgb(255, 255, 255,0.8);
```
Comment (dlhxu) [inline wewatch-client/src/components/Lobby/Lobby.css:17]: nitpick: space after the comma, ',0.8' -> ', 0.8' :)

File: wewatch-client/src/components/Lobby/Lobby.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ const Lobby = ({location}) => {
     return (
         <Layout>
             <div className="outerContainer">
-                <div className="container">
+                <div className= "lobbyContainer">
```
Comment (dlhxu) [inline wewatch-client/src/components/Lobby/Lobby.js:106]: I like how the CSS changes can be applied to both of the major components on this page simulataneously. Good thinking, this is a good application of the DRY principle!

File: wewatch-client/package.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
     "react-scripts": "3.4.4",
     "reactstrap": "^8.7.1",
     "socket.io": "^2.3.0",
-    "styled-components": "^5.2.0"
+    "styled-components": "^5.2.0",
+    "underscore": "^1.12.0"
```
Comment (dlhxu) [inline wewatch-client/package.json:21]: Why did we need to the underscore library here? Was there a hidden bug we didn't catch before?

**Review by billxsheng — APPROVED** [review]
Good job!

File: wewatch-client/package.json (hunk 3 above)
Comment (varoongupta) [reply wewatch-client/package.json:21]: Yeah! I believe that is why I was running into a compile error earlier. I added yarn underscore which seemed to resolve the issue.

File: wewatch-client/src/components/Lobby/Lobby.css (hunk 1 above)
Comment (varoongupta) [reply wewatch-client/src/components/Lobby/Lobby.css]: good catch, updated!

**Review by dlhxu — APPROVED** [review]
Good work!

**Pull request merged**